.buildkite/pipeline.yml

```diff
@@ -5,8 +5,8 @@ steps:
     command: .buildkite/verify-label.sh
   - label: ":k8s:"
     command: .buildkite/verify-rbac-labels.sh
-  - label: ':lipstick:'
-    command: .buildkite/prettier-check.sh
+  #- label: ':lipstick:'
+  #  command: .buildkite/prettier-check.sh
```

Comment:
why is this commented out?

Reply:
temporarily, because yarn was giving me trouble too (on top of atlassian package manager and other stuff). yesterday the software goblins won